rpc_jobs/irr_role_test.yml

```diff
@@ -15,8 +15,37 @@
           IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
       - trusty:
           IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    # NOTE(cloudnull): Context are used to set the OpenStack series used in the gate.
     context:
-      - base
+      - master
+      - ocata
+      - newton
+      - mitaka
+      - liberty
+      - kilo
```

Comment:
This will spin up ~10 15GB instances per PR. That seems a lot. Do any of these instances need to 15gb? Could we reduce the instance size for these jobs?